modules/extension/app-schema/app-schema/pom.xml

```diff
@@ -210,7 +210,19 @@
             <version>${project.version}</version>
             <scope>test</scope>
         </dependency>
-    </dependencies>
+        <dependency>
+             <groupId>org.powermock</groupId>
+             <artifactId>powermock-module-junit4</artifactId>
+             <version>2.0.7</version>
+             <scope>test</scope>
+        </dependency>
+        <dependency>
+             <groupId>org.powermock</groupId>
+             <artifactId>powermock-api-easymock</artifactId>
+             <version>2.0.7</version>
+             <scope>test</scope>
+        </dependency>
```

Comment:
Are this libraries already used in GeoServer? I don't have nothing against PowerMock but we should strive to keep our testing methodology consistent. 

In any case dependencies versions should be declared on the root ``pom.xml``.

Let's see what other have to say about this.

Reply:
Resolved!